DataManagementSystem/Utilities/CatalogUtilities.py

```diff
@@ -0,0 +1,138 @@
+########################################################################
+# $HeadURL$
+########################################################################
+
+""" DIRAC FileCatalog utilities
```

Comment:
They are not really catalog utilities, you use them in the Storage plugin classes